### PR TITLE
K8SPG-64 Test compliance with openshift

### PR DESCRIPTION
**Pull request by cap1984**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| recreate | failed |
| affinity | failed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/66/commits/743797635809b13f43599792abe1fd1167459467
image: `perconalab/percona-postgresql-operator:PR-66-74379763-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-66-74379763-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-66-74379763-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-66-74379763-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-66-74379763-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-66-74379763-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | failed |
| scaling | failed |
| recreate | failed |
| affinity | failed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/66/commits/b9cdf41495477ae1d467b9e9ed0a29fcc07ae54e
image: `perconalab/percona-postgresql-operator:PR-66-b9cdf414-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-66-b9cdf414-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-66-b9cdf414-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-66-b9cdf414-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-66-b9cdf414-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-66-b9cdf414-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| recreate | failed |
| affinity | failed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/66/commits/7f375054f5e0662427be6eb3a2577252ac1f8493
image: `perconalab/percona-postgresql-operator:PR-66-7f375054-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-66-7f375054-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-66-7f375054-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-66-7f375054-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-66-7f375054-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-66-7f375054-pgo-deployer`

File: e2e-tests/functions [hunk 1]
```diff
@@ -84,6 +88,10 @@ deploy_operator() {
 		| $sed -e "s#pgo_image_tag: .*#pgo_image_tag: $(echo ${IMAGE_OPERATOR%-postgres-operator} | cut -d: -f2)#g" \
 			>${tmp_dir}/operator.ini
 
+	if [ ! -z "$OPENSHIFT" ]; then
```
Comment (hors) [inline e2e-tests/functions:91]:
```suggestion
	if [ -n "$OPENSHIFT" ]; then
```

File: e2e-tests/functions [hunk 2]
```diff
@@ -399,6 +407,10 @@ compare_kubectl() {
 	local expected_result=${test_dir}/compare/${resource//\//_}${postfix}.yml
 	local new_result="${tmp_dir}/${resource//\//_}.yml"
 
+	if [ ! -z "$OPENSHIFT" -a -f ${expected_result//.yml/-oc.yml} ]; then
```
Comment (hors) [inline e2e-tests/functions:410]:
```suggestion
	if [ -n "$OPENSHIFT" -a -f ${expected_result//.yml/-oc.yml} ]; then
```

**Review by hors — CHANGES_REQUESTED** [review]

File: e2e-tests/functions (hunk 1 above)
Comment (cap1984) [reply e2e-tests/functions]: done

File: e2e-tests/functions (hunk 2 above)
Comment (cap1984) [reply e2e-tests/functions]: done

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| recreate | failed |
| affinity | failed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/66/commits/0a387f7b928b6ed470fdd71ea42a29c18b69ff02
image: `perconalab/percona-postgresql-operator:PR-66-0a387f7b-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-66-0a387f7b-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-66-0a387f7b-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-66-0a387f7b-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-66-0a387f7b-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-66-0a387f7b-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | failed |
| scaling | failed |
| recreate | failed |
| affinity | failed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/66/commits/81dcb18649730390a1755afaf1598240bb30077a
image: `perconalab/percona-postgresql-operator:PR-66-81dcb186-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-66-81dcb186-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-66-81dcb186-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-66-81dcb186-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-66-81dcb186-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-66-81dcb186-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | failed |
| scaling | failed |
| recreate | failed |
| affinity | failed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/66/commits/825f7c42c67b7269c8e486af41cb9d84f7e109fe
image: `perconalab/percona-postgresql-operator:PR-66-825f7c42-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-66-825f7c42-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-66-825f7c42-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-66-825f7c42-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-66-825f7c42-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-66-825f7c42-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | failed |
| scaling | failed |
| recreate | failed |
| affinity | failed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/66/commits/5331af3b5f33f8de7402ebd404b7daf4afd8f9ed
image: `perconalab/percona-postgresql-operator:PR-66-5331af3b-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-66-5331af3b-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-66-5331af3b-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-66-5331af3b-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-66-5331af3b-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-66-5331af3b-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | failed |
| scaling | failed |
| recreate | failed |
| affinity | failed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/66/commits/eaa8e570127afd12bd57179d2aaa3cbef206d809
image: `perconalab/percona-postgresql-operator:PR-66-eaa8e570-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-66-eaa8e570-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-66-eaa8e570-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-66-eaa8e570-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-66-eaa8e570-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-66-eaa8e570-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | failed |
| scaling | failed |
| recreate | failed |
| affinity | failed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/66/commits/eaa8e570127afd12bd57179d2aaa3cbef206d809
image: `perconalab/percona-postgresql-operator:PR-66-eaa8e570-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-66-eaa8e570-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-66-eaa8e570-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-66-eaa8e570-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-66-eaa8e570-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-66-eaa8e570-pgo-deployer`

**Pull request closed without merging**